src/test/rpc/LedgerRPC_test.cpp

```diff
@@ -1724,11 +1732,18 @@ class LedgerRPC_test : public beast::unit_test::suite
         testLedgerEntryPayChan();
         testLedgerEntryRippleState();
         testLedgerEntryTicket();
-        testLedgerEntryUnknownOption();
         testLookupLedger();
         testNoQueue();
         testQueue();
         testLedgerAccountsOption();
+
+        // version specific tests
+        for (auto testVersion = RPC::apiMinimumSupportedVersion;
+             testVersion <= RPC::apiBetaVersion;
+             ++testVersion)
+        {
+            testLedgerEntryInvalidParams(testVersion);
+        }
```

Comment:
nit extra bracket

Reply:
Kept it for readability. It was becoming hard to read after the indentation caused by clang-format if I removed the brackets.